### ISSUE TITLE
White buttons in Activity when using dark theme with AppCompat 22

### ISSUE THREAD

**Issue by ultrah** [fengdai/AlertDialogPro#43]
When using Theme.AlertDialogPro.Holo as the theme of an activity, all buttons have a white background and white text when the app also uses AppCompat 22.

With AppCompat 21 it looks fine. I was able to reproduced this from Android 2.3 to 4.4.4

![white_buttons](https://cloud.githubusercontent.com/assets/492486/6768953/aaa81d6c-d085-11e4-8603-fa2504465157.PNG)

**Comment by fengdai** [comment]
Theme.AlertDialogPro.Holo is designed to be used as the theme of AlertDialogPro. I can't guarantee that it can be used well as the theme of an activity.

**Comment by ultrah** [comment]
Makes sense. I  used it for convenience, to align the look with the dialogs created by AlertDialogPro. I'll just switch to Theme.AppCompat.Dialog for activities.
